packit/config/job_config.py

```diff
@@ -95,6 +97,7 @@ def __init__(
         self.fmf_ref: str = fmf_ref
         self.use_internal_tf: bool = use_internal_tf
         self.skip_build: bool = skip_build
+        self.env: Dict[str, Any] = env or {}
```

Comment:
Can't we use `Dict[str,str]`?

Reply:
I think that a user may want to pass in an `int`, e.g. to achieve the same as `FOO=5` which should be valid. That's why I went with the non-restrictive Any, even though string and int are probably the main examples I can think of.